docs/Attributes/GetCollection.md

```diff
@@ -43,6 +47,9 @@ final class DummyQueryInput
 You could override those or add more information using the following parameters
 
 ```php
+use ApiScout\Attribute\GetCollection;
+use ApiScout\Attribute\ApiProperty;
+
     #[GetCollection(
```

Comment:
It's not very pretty as it is, since the use have been demonstrated already maybe just remove them here ?

Reply:
I understand, but for my part, I prefer it when a code example is self-contained and not dependent on a preceding block.

But it's your tool, so I'll remove these two lines if you prefer